name.abuchen.portfolio/src/name/abuchen/portfolio/datatransfer/pdf/WirBankPDFExtractor.java

```diff
@@ -79,7 +79,7 @@ private void addBuySellTransaction()
             return entry;
         });
 
-        Block firstRelevantLine = new Block("^(B.rsenabrechnung|Exchange Settlement) \\- (Kauf|Verkauf|Buy|Sell)$");
```

Comment:
Please replace `(\\s.*)?` to `.*`
This implements the same like "zero or more" and is cleaner source.

Reply:
Technically, the statement says "sometimes it is followed by a space and then more arbitrary characters". Therefore the statements are not identical.

When checking against the PDF extract, the .* is probably still applicable because we have the four explicit expression. I will simplify and merge.